pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -496,6 +496,12 @@ func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.Servic
 		replica = service.Replicas
 	}
 
+	// Check to see if Docker Compose v3 Deploy.Mode has been set to "global"
+	if service.DeployMode == "global" {
+		log.Infof("Deploy mode has been set to global. Setting Kubernetes container replicas to 1 for service %s", name)
```

Comment:
By default DeployMode is "replicated" so this will only be shown when "global" is selected. Should we remove this all-together?